test/fuzz/signature/signature.go

```diff
@@ -57,3 +60,88 @@ func FuzzECDSASigner(data []byte) int {
 
 	return 1
 }
+
+func FuzzComputeDigest(data []byte) int {
+	ecdsaSupportedHashFuncs := []crypto.Hash{
+		crypto.SHA256,
+		crypto.SHA512,
+		crypto.SHA384,
+		crypto.SHA224,
+		crypto.SHA1,
+	}
+	data, _, err := signature.ComputeDigestForSigning(bytes.NewReader(data), crypto.SHA512, ecdsaSupportedHashFuncs)
+	if err != nil {
+		if data != nil {
+			panic(fmt.Sprintf("key %v is not nil when there is an error %v ", data, err))
+		}
+		return 0
+	}
+	return 1
+}
+
+func FuzzComputeVerifying(data []byte) int {
+	ecdsaSupportedHashFuncs := []crypto.Hash{
```

Comment:
maybe a more generic name since there's no relation to ecdsa here?

Reply:
Agreed. Thanks